src/kvstore/raftex/RaftPart.h

```diff
@@ -539,6 +539,15 @@ class RaftPart : public std::enable_shared_from_this<RaftPart> {
     std::shared_ptr<folly::IOThreadPoolExecutor> ioThreadPool_;
     // Shared worker thread pool
     std::shared_ptr<thread::GenericThreadPool> bgWorkers_;
+    // In order to improve the stability of the heartbeat, each leader holds a separate thread
+    // to send its heartbeat.
+    // Note:
+    //  - leaderHeartbeatWorker_ only works when the role is leader, and other roles use the
+    //    shared bgWorker_.
+    // Problem solved:
+    //  - When the write load is high and the disk IO Util is high, shared bgworker_ may be too
+    //    busy, causing more frequent follower election.
+    thread::GenericThreadPool leaderHeartbeatWorker_;
```

Comment:
Yeah, this may overcorrect. The reason is that the disk IO is full and bgwoker is too busy.It is also possible to continue to increase the size of bgwoker.